### PR TITLE
update references

### LINKED ISSUE #1009: update for oops PR, Update the UnstructuredInterpolator's geometric algorithms
make sure soca is up to date with upstream PR 
- https://github.com/JCSDA-internal/oops/pull/2532#event-11612220828

bonus points for actually testing it with the global 1/4 deg

### PR DESCRIPTION
**Pull request by travissluka**
## Description
- closes #1009 

## Testing

requires matching branches in saber and oops

- [x] ctests pass
- [x] test with HAT10 (I ran 2 cycles, seems okay)
- [ ] test with global 1/4

## Dependencies

build-group=https://github.com/JCSDA-internal/saber/pull/758
build-group=https://github.com/JCSDA-internal/oops/pull/2532

### CODE REVIEW

File: test/testinput/3dvar_godas.yml [hunk 1]
```diff
@@ -276,3 +276,4 @@ final:
 test:
   reference filename: testref/3dvar_godas.test
   test output filename: testoutput/3dvar_godas.test
+  float relative tolerance: 1e-5
```
Comment (fmahebert) [inline test/testinput/3dvar_godas.yml:279]: Did you find that the new code is more machine-dependent than develop?

Comment (travissluka) [reply test/testinput/3dvar_godas.yml:279]: @fmahebert it's hard to say, this is the only place where the tolerances were increased, what was odd though was that the gnu CI test here needed the tolerance increase (and  I develop with GCC 12 at home). It's entirely possible this test was close to exceeding the tolerance anyway and we never noticed.

Comment (fmahebert) [reply test/testinput/3dvar_godas.yml:279]: Yeah, fair enough. In fv3-jedi a lot of the variational tests have very loose tolerances, too.

**Review by fmahebert — APPROVED** [review]
CI approves (except intel which looks like it failed from some network/authentication glitch), so I transitively approve. Plus, the ref changes look reasonable.

**Review by kbhargava — APPROVED** [review]
Thank for updating!! looks good.

**Comment by ytremolet** [comment]
@travissluka The OOPS PR has been merged.

**Pull request merged**